pkg/container/container.go

```diff
@@ -182,6 +182,8 @@ func (c *container) SpecAddAnnotations(ctx context.Context, sb *sandbox.Sandbox,
 				// Annotation not suffixed with the container name
 				c.spec.AddAnnotation(k, v)
 			}
+		} else {
+			c.spec.AddAnnotation(k, v)
```

Comment:
Let's simplify the code by `continue`'ing after line 180 and removing the `} else {`s to just `c.spec.AddAnnotation(k, v)` in all other cases.

Reply:
Respectfully, I think such a change makes the code less readable, i.e. harder for a human to understand what's happening.  I think balanced if/else clauses are the way to go.

Reply:
@saschagrunert alternatively I could simplify the whole thing to something like:
```go
for k, v := range sb.Annotations() {
  if k == crioann.OCISeccompBPFHookAnnotation + "/" + c.config.Metadata.Name {
    log.Debugf(ctx,
	       "Annotation key for container %q rewritten to %q (value is: %q)",
	       c.config.Metadata.Name, crioann.OCISeccompBPFHookAnnotation, v,
    )
    c.config.Annotations[crioann.OCISeccompBPFHookAnnotation] = v
    c.spec.AddAnnotation(crioann.OCISeccompBPFHookAnnotation, v)
  } else {
    c.spec.AddAnnotation(k, v)
  }
}
```

(Preserving the existing comments of course).  What do you think?

Reply:
Yeah sound good to me, thank you for the alternative appraoch.